bin/update-reproducibility-summary.sh

```diff
@@ -132,15 +143,15 @@ do
         echo "Source code: [$gitRepo]($gitRepo)" >> ${projectReadme}
         echo >> ${projectReadme}
 
-        projectGa="$(( $(cat $dir/*.buildinfo | grep coordinates | cut -d = -f 2 | sort -u | wc -l) ))"
+        projectGa="$(( $(cat $dir/*-${newestBuildinfoVersion}\.buildinfo | grep coordinates | cut -d = -f 2 | sort -u | wc -l) ))"
         if [ $projectGa -gt 1 ]
         then
           echo "<details><summary>This project defines $projectGa modules:</summary>" >> ${projectReadme}
           echo >> ${projectReadme}
-          for ga in $(cat $dir/*.buildinfo | grep coordinates | cut -d = -f 2 | sort -u)
+          for ga in $(cat $dir/*-${newestBuildinfoVersion}\.buildinfo | grep coordinates | cut -d = -f 2 | sort -u)
           do
             gaDir=$(echo "$ga" | sed -e 's_:_/_')
-            echo "* [$ga](https://central.sonatype.com/artifact/${gaDir}/${version})" >> ${projectReadme}
+            echo "* [$ga](https://central.sonatype.com/artifact/${gaDir}/${newestBuildinfoVersion})" >> ${projectReadme}
```

Comment:
IIUC, this will apply the same version for each and every module: the issue is that some modules disappeared, some were added, then there is no single value matching all modules, it would have to be per-module

checking again, as of now, I can just use the `overview` url, without any version: it matches what we need IMHO, it was just not there when the feature was added

I'll switch to the non-versioned url